Makefile

```
@@ -366,7 +366,7 @@ set-authorino-default-image: yq ## Sets the default Authorino image in the build
 set-replaces-directive: yq ## Sets the value for the OLM replaces directive in the build file.
 	$(eval REPLACES_VERSION=$(shell curl -sSL -H "Accept: application/vnd.github+json" \
                https://api.github.com/repos/Kuadrant/authorino-operator/releases/latest | \
```

checkmake (0.3.2)

[warning] Target "https" should be declared PHONY.

(phonydeclared)